pom.xml

```diff
@@ -1360,6 +1508,11 @@
     <module>fractions/keycloak</module>
     <module>fractions/microprofile</module>
     <module>fractions/monitor</module>
+    <module>fractions/netflix</module>
+    <module>fractions/nosql/cassandra</module>
+    <module>fractions/nosql/mongodb</module>
+    <module>fractions/nosql/neo4j</module>
+    <module>fractions/nosql/orientdb</module>
```

Comment:
What is the purpose of the unsupported profile?  Is that for experimental fractions, like the NoSQL fractions?  Or is that more of a way to indicate that the fractions are not expected to be in product yet?

Easy enough for me to move them to the unsupported profile.  I would like to develop community around the NoSQL fractions and see them move to be supported at some point. :)

I'll remove the extra netflix module as well.